doc/changelog/09-coqide/14696-alizter+coqide+name.rst

```diff
@@ -0,0 +1,5 @@
+- **Changed:**
+  All occurrences of the name `CoqIde` to `CoqIDE`.
```

Comment:
Please add a note that this may result in program icons being duplicated instead of replaced on some systems when updating (without this warning, the changelog isn't worth much).